weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx

```diff
@@ -37,6 +54,34 @@ export const CallSummary: React.FC<{
   );
   const costData = call.traceCall?.summary?.weave?.costs;
 
+  const storageSizeBytesRow = useMemo(() => {
+    const size =
+      getJsonPayloadSize(span.inputs) +
```

Comment:
if you are determining size with these properties, why do you need to request the size from the server?